c/zss.c

```diff
@@ -93,6 +93,26 @@ static JsonObject *readPluginDefinition(ShortLivedHeap *slh, char *pluginIdentif
 static WebPluginListElt* readWebPluginDefinitions(HttpServer* server, ShortLivedHeap *slh, char *dirname);
 static JsonObject *readServerSettings(ShortLivedHeap *slh, const char *filename);
 static InternalAPIMap *makeInternalAPIMap();
+ 
+ //redo
+  static int keyCompare(void *key1, void *key2) {
+  if (strcmp(key1,key2) == 0){
+    return TRUE;
+  }  
+  return FALSE;
+}
+/*
+// copied from collections
```

Comment:
What hashing function are you using now?

Reply:
the function already exists in collections.c, so instead of copying it into zss.c i use the one that already exists.